Packs/Tanium/Integrations/Tanium_v2/Tanium_v2.yml

```diff
@@ -1235,6 +1235,8 @@ script:
     - defaultValue: via Demisto API
       description: The action name.
       name: action-name
+    - description: Expiration time (in seconds) for the package.
+      name: expire-in-seconds
```

Comment:
```suggestion
      name: expire_time 
```

Reply:
Changed to `expiration-time` as we discussed.